tox.ini

```diff
@@ -47,9 +47,9 @@ allowlist_externals =
 commands_pre =
     pip-sync requirements/{env:TOX_ENV_NAME}.txt --pip-args '--disable-pip-version-check'
 commands =
-    dev: {posargs:supervisord -c conf/supervisord-dev.conf}
```

Comment:
Apparently this line got moved further down the file in the cookiecutter :man_shrugging: